snow/consensus/snowman/consensus_test.go

```diff
@@ -1171,6 +1172,88 @@ func RecordPollChangePreferredChainTest(t *testing.T, factory Factory) {
 	require.False(sm.IsPreferred(b2Block))
 }
 
+func LastAcceptedTest(t *testing.T, factory Factory) {
+	sm := factory.New()
+	require := require.New(t)
+
+	ctx := snow.DefaultConsensusContextTest()
+	params := snowball.Parameters{
+		K:                     1,
+		Alpha:                 1,
+		BetaVirtuous:          1,
+		BetaRogue:             2,
+		ConcurrentRepolls:     1,
+		OptimalProcessing:     1,
+		MaxOutstandingItems:   1,
+		MaxItemProcessingTime: 1,
+	}
+	require.NoError(sm.Initialize(ctx, params, GenesisID, GenesisHeight, GenesisTimestamp))
+
+	block0 := &TestBlock{
+		TestDecidable: choices.TestDecidable{
+			IDV:     ids.GenerateTestID(),
+			StatusV: choices.Processing,
+		},
+		ParentV: Genesis.IDV,
+		HeightV: Genesis.HeightV + 1,
+	}
+	block1 := &TestBlock{
+		TestDecidable: choices.TestDecidable{
+			IDV:     ids.GenerateTestID(),
+			StatusV: choices.Processing,
+		},
+		ParentV: block0.IDV,
+		HeightV: block0.HeightV + 1,
+	}
+	block2 := &TestBlock{
+		TestDecidable: choices.TestDecidable{
+			IDV:     ids.GenerateTestID(),
+			StatusV: choices.Processing,
+		},
+		ParentV: block1.IDV,
+		HeightV: block1.HeightV + 1,
+	}
+	block1Conflict := &TestBlock{
```

Comment:
Is this variable useful for this test?

Reply:
Removing `block1Conflict` will cause the test to fail... I added it because I thought it made sense to add something getting rejected to the test